base/sparse/cholmod.jl

```diff
@@ -1750,17 +1752,22 @@ end
 
 det(L::Factor) = exp(logdet(L))
 
-function isposdef(A::SparseMatrixCSC{<:VTypes,SuiteSparse_long})
-    if !ishermitian(A)
-        return false
-    end
-    try
-        f = cholfact(A)
-    catch e
-        isa(e, LinAlg.PosDefException) || rethrow(e)
+function issuccess(F::Factor)
+    s = unsafe_load(F.p)
+    return s.minor == size(F, 1)
+end
+
+function isposdef(F::Factor)
+    if issuccess(F)
+        s = unsafe_load(F.p)
+        if s.is_ll == 1
+            return true
+        else
+            return :maybe
```

Comment:
I'm pretty sure it is but a lot of stuff was still broken so there was many test failures. I'll take a look at the coverage when tests pass.